tools/convert.rb

```diff
@@ -1239,8 +1239,8 @@ def checkRightsOverride(unitID, volNum, issNum, oldRights)
       # Failing that, check for a default set on the unit
       unit = $allUnits[unitID]
       unitAttrs = unit && unit.attrs && JSON.parse(unit.attrs) || {}
-      if unitAttrs["default_issue"]
-        rights = unitAttrs["default_issue"]
+      if unitAttrs["default_issue"] && unitAttrs["default_issue"]["rights"]
```

Comment:
@lschiff FYI checking this in

Reply:
Btw this was me not Lisa, but no matter.